ui/src/app/components/cases/test-case-details.component.ts

```diff
@@ -126,10 +126,10 @@ export class TestCaseDetailsComponent extends BaseComponent implements OnInit {
         data: {
           description: this.translate.instant("message.common.confirmation.message", {FieldName: this.isGroup }),
           isPermanentDelete: permanently,
-          title: 'Test Case',
-          item: 'test case',
+          title: this.isGroup,
```

Comment:
using is as prefix for variable usually refers to boolean constraint, plz have a better naming

Reply:
done, have replaced isGroup with checkCaseType. check latest commit